### DISCUSSION TITLE
Current smart plugs that can be flashed with Tasmota

### DISCUSSION THREAD

**Discussion by GrantsPi** [arendst/Tasmota#17158] [Q&A] [answered]
Where can I find a list of smart plugs with power monitoring that can still be flashed with Tasmota.  I have a number of older plugs that work well and want to buy more plugs with power monitoring that I can flash with Tasmota

**Comment by barbudor** [comment, marked as answer]
The template site is still the only reference that we have.
It's community based so if no-one come to update the description ...
A safe alternative is to pickup in the list of devices that are sold pre-flashed with Tasmota

**Reply by GrantsPi** [reply]
Great!  Thank you.
